containers/ecr-viewer/src/app/admin/user/UserForm.tsx

```diff
@@ -0,0 +1,265 @@
+  programs: FormProgram[];
+}
+
+/**
+ *
+ * @param props React props
+ * @param props.initValues Initial values the form is set to
+ * @param props.action Action of the form (e.g. "Create", "Edit")
+ * @param props.submitAction Handler for the submitted data
+ * @returns Program area add/edit form
+ */
+export const UserForm = ({
+  action,
+  initValues,
+  submitAction,
+}: {
+  action: string;
+  initValues: FormValues;
+  submitAction: (
+    email: string,
+    userType: UserType,
+    programs: string[],
+  ) => Promise<ServerActionResult<string>>;
+}) => {
+  const [email, setEmail] = useState(initValues.email || "");
+  const [userType, setUserType] = useState<UserType>("standard");
+  const [programs, setPrograms] = useState(initValues.programs);
+
+  const { createToast } = React.useContext(ToastContext);
+
+  const selectedPrograms = Object.values(programs)
+    .filter(({ checked }) => !!checked)
+    .map(({ uuid }) => uuid);
+  const numProgramsSelected = selectedPrograms.length;
+
+  const valid = !!email && numProgramsSelected > 0;
+
+  return (
+    <FormPageContent
+      action={`${action} user`}
+      formValid={valid}
+      submitAction={async () => {
+        const res = await submitAction(
+          email.trim(),
+          userType,
+          selectedPrograms,
+        );
+        if (!res.error)
+          createToast(`${email.trim()} successfully saved`, "success");
+        return res;
+      }}
+      successRoute="/admin/user"
+    >
+      <EmailFieldSet email={email} setEmail={setEmail} />
+      <UserTypeFieldSet userType={userType} setUserType={setUserType} />
+      <ProgramFieldSet
+        programs={programs}
+        setPrograms={setPrograms}
+        numProgramsSelected={numProgramsSelected}
+      />
+    </FormPageContent>
+  );
+};
+
+const EmailFieldSet = ({
+  email,
+  setEmail,
+}: {
+  email: string;
+  setEmail: (n: string) => void;
+}) => {
+  return (
+    <FieldSet legend="Email">
+      <span>
+        Add the new user by their login email (<RequiredMarker />)
+      </span>
+      <label className="usa-label">
+        EMAIL
+        <RequiredMarker />
+        <TextInput
+          type="email"
+          required={true}
+          id="email"
+          name="email"
+          value={email}
+          onChange={(e) => setEmail(e.target.value)}
+        />
+      </label>
+    </FieldSet>
+  );
+};
+
+const UserTypeFieldSet = ({
+  userType,
+  setUserType,
+}: {
+  userType: UserType;
+  setUserType: (n: UserType) => void;
+}) => {
+  return (
+    <FieldSet legend="Select user type">
+      <span>
+        User type will apply to all users added. These can be edited later (
+        <RequiredMarker />)
+      </span>
+      <label className="usa-label">
+        USER TYPE
+        <RequiredMarker />
+        {["admin", "standard"].map((option) => (
+          <Radio
+            key={option}
+            label={toTitleCase(option)}
+            type="radio"
+            required={true}
```

Comment:
the design has some explainer on each user type - is adding that in a follow on ticket already? or should it be done here?

<img width="621" alt="image" src="https://github.com/user-attachments/assets/dd5446ce-bee1-475c-8c7b-c224e3eb7354" />